tests/libs/key-event.js

```diff
@@ -18,7 +18,14 @@
     };
 
     KeyEvent.prototype.toNative = function() {
-        var event = document.createEventObject ? document.createEventObject() : document.createEvent('Events');
+        var event;
+
+        // Events created with createEventObject() in IE 10 don't work with setting properties like `which`, etc.
+        if (/*@cc_on!@*/false && document.documentMode === 10) {
```

Comment:
It executes in IE
https://stackoverflow.com/questions/24473882/what-does-this-comment-cc-on-0-do-inside-an-if-statement-in-javascript/24473897#24473897